nixos/modules/services/misc/languagetool.nix

```diff
@@ -0,0 +1,82 @@
+{ config, lib, options, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.services.languagetool;
+in {
+  options.services.languagetool = {
+    enable = mkOption {
+      type = types.bool;
+      example = true;
+      default = false;
+      description = mdDoc ''
+        Start the HTTP LanguageTool server.
+      '';
+    };
```

Comment:
```suggestion
    enable = mkEnableOption "the LanguageTool server";
```

Reply:
should this be
```
enable = mkEnableOption (mdDoc "the LanguageTool server");
```
?

Reply:
no, `mkEnableOption` does not support mdDoc at the moment. we're working on it :)